examples/hello_world/CMakeLists.txt

```diff
@@ -3,33 +3,37 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
-cmake_minimum_required (VERSION 2.8.7)
+cmake_minimum_required (VERSION 3.13)
 project (vSomeIPHelloWorld)
 
 find_package(Threads REQUIRED)
 
```

Comment:
Don't you also need a `find_package(vsomeip3)` here?

I know that line exists below in the Android-NDK section, which also has never made sense to me (since both the NDK and non-NDK builds need it)